### PR TITLE
Update admin-cli.md 

### PR DESCRIPTION
**Pull request by lipingxue**
Update user guide (admin-cli.md) accordingly after the DEFAULT_TENANT, DEFAULT_PRIVILEGE, and DEFAULT_DATASTORE change are introduced.

- update existing samples with the latest command output
- add a section about new command "tenant update"
- update the description of command "tenant access add" regarding the "default_datastore" management.

### CODE REVIEW

**Comment by lipingxue** [comment]
@pdhamdhere @kerneltime Please review it.

File: docs/user-guide/admin-cli.md [hunk 1]
```diff
@@ -20,68 +20,74 @@ of their usage.
...
+    access              Add or remove Datastore access and quotas for a tenant
+    ls                  List tenants and the VMs they are applied to
+    rm                  Delete a tenant
 
 optional arguments:
   -h, --help            show this help message and exit
 ```
 
 ### Create
+A tenant named "_DEFAULT" should have been created by system automatically before users try to create any tenant.
```
Comment (kerneltime) [inline docs/user-guide/admin-cli.md]: A tenant named "_DEFAULT" should have been created by system automatically before users try to create any tenant. => A tenant named "_DEFAULT"  will be created automatically post install.

**Comment by kerneltime** [comment]
We need a small write up of the default tenant behavior [here](https://github.com/vmware/docker-volume-vsphere/blob/890415d85af970e9e7c25cd86b254d8d3d907be7/docs/features/tenancy.md) 
It should explain that default tenant is created.. what is the behavior for default tenant for datastores, when and what should a VM that gets assigned to default tenant expect..

**Comment by kerneltime** [comment]
Will continue to review.

File: docs/user-guide/admin-cli.md [hunk 2]
```diff
@@ -93,35 +99,100 @@ optional arguments:
   -h, --help  show this help message and exit
 ```
 
+### Update
+Update existing tenant. This command allows to update "Description", "Default_datastore" or rename an existing tenant.
```
Comment (msterin) [inline docs/user-guide/admin-cli.md]: This command allows to update "Description" and "Default_datastore" fields, or rename an existing tenant.

File: docs/user-guide/admin-cli.md [hunk 3]
```diff
@@ -266,69 +370,41 @@ Set command allows to change the existing access control in place for a tenant.
...
-                        Maximum number of volumes to create on the datastore
-                        for this tenant
-  --rm-rights create,delete,mount,all
-                        Datastore access Permissions removed: Choices =
-                        ['create', 'delete', 'mount', 'all']
-  --add-rights create,delete,mount,all
-                        Datastore access Permissions granted: Choices =
-                        ['create', 'delete', 'mount', 'all']
-  --name NAME           Tenant name
+  --allow-create        Allow create and delete on datastore if set to True
```
Comment (msterin) [inline docs/user-guide/admin-cli.md]: `the` datastore to be consistent with other help

**Comment by lipingxue** [comment]
@kerneltime 
Mark has a detailed design about default_tenant, default_privilege and default_datastore here. https://github.com/vmware/docker-volume-vsphere/issues/621
Please take a look. Should we just put a link to this document  in tenancy.md or we can write a short paragraph based on this document.

**Comment by lipingxue** [comment]
Address comments fro Ritesh. A small write up of the default tenant behavior is added.

File: docs/features/tenancy.md [hunk 4]
```diff
@@ -17,6 +17,21 @@ limits at granularity of datastore.
 
 Tenants can be created and managed via the [Admin CLI](/user-guide/admin-cli/#tenant)
 
+## Default tenant
+When a VM which does not belong to any tenant issues a request to vmdk_ops, this VM will be assumed to be in _DEFAULT tenant, and will get privileges 
+associated with this tenant. \_DEFAULT tenant will be automatically created by system post install, so by default vmdk_ops will support request from
```
Comment (kerneltime) [inline docs/features/tenancy.md]: typo?

File: docs/user-guide/admin-cli.md [hunk 5]
```diff
@@ -266,69 +370,41 @@ Set command allows to change the existing access control in place for a tenant.
...
+datastore1  False         500.00MB         1.00GB      
+
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant access set --name=tenant1 --datastore=datastore1 --allow-create --volume-maxsize=1000MB --volume-totalsize=2GB
 tenant access set succeeded
-[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant access ls --name tenant1
-Datastore   Create_volume  Delete_volume  Mount_volume  Max_volume_size  Total_size
-----------  -------------  -------------  ------------  ---------------  ----------
-datastore1  1              1              1             500.00GB         2.00TB
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant access ls --name=tenant1
+Datastore   Allow_create  Max_volume_size  Total_size
```
Comment (kerneltime) [inline docs/user-guide/admin-cli.md]: White space at the end

File: docs/user-guide/admin-cli.md [hunk 6]
```diff
@@ -266,69 +370,41 @@ Set command allows to change the existing access control in place for a tenant.
...
+
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant access set --name=tenant1 --datastore=datastore1 --allow-create --volume-maxsize=1000MB --volume-totalsize=2GB
 tenant access set succeeded
-[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant access ls --name tenant1
-Datastore   Create_volume  Delete_volume  Mount_volume  Max_volume_size  Total_size
-----------  -------------  -------------  ------------  ---------------  ----------
-datastore1  1              1              1             500.00GB         2.00TB
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant access ls --name=tenant1
+Datastore   Allow_create  Max_volume_size  Total_size  
+----------  ------------  ---------------  ----------
```
Comment (kerneltime) [inline docs/user-guide/admin-cli.md]: Here as well

File: docs/user-guide/admin-cli.md [hunk 7]
```diff
@@ -20,68 +20,74 @@ of their usage.
...
 #### Help
 ```
-[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant create -h
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant create -h
 usage: vmdkops_admin.py tenant create [-h] --name NAME
+                                      [--description DESCRIPTION]
                                       [--vm-list vm1, vm2, ...]
 
 optional arguments:
   -h, --help            show this help message and exit
```
Comment (shuklanirdesh82) [inline docs/user-guide/admin-cli.md]: Nit: show => Show (to keep it consistent with the rest)

File: docs/user-guide/admin-cli.md [hunk 8]
```diff
@@ -93,35 +99,100 @@ optional arguments:
...
+#### Help
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant update -h
+usage: vmdkops_admin.py tenant update [-h] --name NAME
+                                      [--default-datastore DEFAULT_DATASTORE]
+                                      [--description DESCRIPTION]
+                                      [--new-name NEW_NAME]
+
+optional arguments:
+  -h, --help            show this help message and exit
```
Comment (shuklanirdesh82) [inline docs/user-guide/admin-cli.md]: nit: same as above

File: docs/user-guide/admin-cli.md [hunk 9]
```diff
@@ -93,35 +99,100 @@ optional arguments:
...
+527e94ec-43e9-4d78-81fe-d99ab06a54b3  _DEFAULT  This is a default tenant                              
+```
+
+#### Help
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant rm -h
+usage: vmdkops_admin.py tenant rm [-h] --name NAME [--remove-volumes]
+
+optional arguments:
+  -h, --help        show this help message and exit
```
Comment (shuklanirdesh82) [inline docs/user-guide/admin-cli.md]: nit: same as above

**Pull request merged**